.gitignore

```diff
@@ -1 +1,4 @@
 node_modules
+cloud-init.yaml
+id_ed25519
+id_ed25519.pub
```

Comment:
you can do id_ed25519* to handle both of these files by using a filename wildcard *